bin/node-template/node/src/service.rs

```diff
@@ -78,14 +83,31 @@ pub fn new_partial(config: &Configuration) -> Result<sc_service::PartialComponen
 	})
 }
 
+fn remote_keystore(_url: &String) -> Result<Arc<LocalKeystore>, &'static str> {
```

Comment:
How is a remote store going to return a concrete `LocalKeystore`? Shouldn't this be `Arc<dyn CryptoStore>`?

Reply:
We face the same `CryptoStore + SyncCryptoStore` interface problem here, so I can't quite make this a `dyn` unless I provide a similar wrapper. Which is quite the overkill as in reality this will always return a concrete type and thus the interface is unnecessary. In this example I just used the local-store as the concrete type (that, as you can see, is never returned)